sdk/nodejs/image.ts

```diff
@@ -46,6 +46,10 @@ export interface ImageArgs {
      * Credentials for the docker registry to push to.
      */
     registry?: pulumi.Input<ImageRegistry>;
+    /**
+     * Skip push flag.
+     */
+    skipPush?: pulumi.Input<boolean>;
```

Comment:
This should also be a `boolean` :)

Reply:
It's being checked on:
const skipPush = (imageArgs.skipPush && imageArgs.skipPush === true) ? true : false;
So your case with Output<boolean> should be catched here now. But will change as well.